kivy/lib/osc/OSC.py

```diff
@@ -177,24 +231,45 @@ def OSCArgument(data):
     """Convert some Python types to their
     OSC binary representations, returning a
     (typetag, data) tuple."""
+        
+    try:
```

Comment:
There's an issue if data being given isn't valid; probably poor coding practices on my own end as I don't wrap the sending method to catch error data.
This closer matches incorrect data coming from on the TCP connection though.